mathics/builtin/numbers/algebra.py

```diff
@@ -767,11 +848,28 @@ class Expand(_Expand):
     #> (y^2)^(1/2)/(2x+2y)//Expand
      = Sqrt[y ^ 2] / (2 x + 2 y)
 
-    ## This caused a program crash!
```

Comment:
@mmatera Do we still want this comment?

Reply:
I do not think so, since it pass the test...